github/files/drupal7/scripts/devops/includes/reinstall/pre_install_tasks.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Creating cache folder
```

Comment:
@Sanchiz It would be nice to have documentation aka file dockblock within every include